src/player.cpp

```diff
@@ -7756,6 +7756,18 @@ bool player::wield( item& target )
     if( target.is_null() ) {
         return true;
     }
+
+    // Query whether to draw an item from a holster when attempting to wield the holster
+    if( target.get_use( "holster" ) && ( target.contents.size() > 0 ) )
+    {
```

Comment:
I don't want to be annoying and don't want to discourage you from other PRs but it still wrong formatted. ```{``` should be on a previous line (see an example right above), and